server/models/LevelSession.coffee

```diff
@@ -81,7 +81,7 @@ LevelSessionSchema.pre 'save', (next) ->
     next()
 
 LevelSessionSchema.statics.privateProperties = ['code', 'submittedCode', 'unsubscribed']
-LevelSessionSchema.statics.editableProperties = ['multiplayer', 'players', 'code', 'codeLanguage', 'completed', 'state',
+LevelSessionSchema.statics.editableProperties = ['multiplayer', 'players', 'code', 'codeLogs', 'codeLanguage', 'completed', 'state',
```

Comment:
Should remove `codeLogs` from this list, that way we know that property only gets created on POSTing a code log.